components/wpt-results.html

```diff
@@ -194,11 +194,12 @@
             type: Boolean,
             computed: '_computeIsLatest(sha)'
           },
-          // ['browser@sha] spec for the runs to load
+          // ['browser@sha', ...] specs for the runs to load using /api/run
+          // e.g. ['chrome@latest', 'edge@latest', 'firefox@latest', 'safari@latest']
           testRunSpecs: {
             type: Array
           },
-          // JSON blobs for the runs loaded (by their specs)
+          // Fetched + parsed JSON blobs for the runs loaded (by their specs, above)
```

Comment:
Is the structural type inside the Array small enough to describe it here?

Reply:
Is it standard practice to bloat source-code with comments of the format that the data takes? Shouldn't that (self-)documentation live somewhere else?

Reply:
Yes, and yes. The trouble is that we don't currently have protos or JSON schema, or `.md` files, or anything for this stuff. If you can use a type name or a link, I'm all for it, but I don't think such types/links exist for these data.

Perhaps it's best to just file (and/or pursue existing?) bugs about structuring the data.